src/Symfony/Component/Amqp/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "symfony/amqp",
+    "type": "library",
+    "description": "Library to dialog with AMQP",
+    "keywords": ["amqp", "rabbitmq", "consumer", "producer", "queue", "exchange"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Grégoire Pineau",
+            "email": "lyrixx@lyrixx.info"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3",
+        "ext-amqp": ">=1.5",
+        "psr/log": "~1.0",
+        "symfony/event-dispatcher": "^2.3|^3.0|^4.0"
```

Comment:
should be `^3.4|^4.0` as in other components (we bump these at each major versions)